src/GafferImage/Catalogue.cpp

```diff
@@ -178,6 +190,7 @@ class Catalogue::InternalImage : public ImageNode
 			fileNamePlug()->source<StringPlug>()->setValue( other->fileNamePlug()->getValue() );
 			imageSwitch()->indexPlug()->setValue( other->imageSwitch()->indexPlug()->getValue() );
 			text()->enabledPlug()->setValue( other->text()->enabledPlug()->getValue() );
+			setIsRendering( other->getIsRendering() );
```

Comment:
I don't think a copy should be considered to be rendering should it? The code below takes snapshots of the current driver state rather than remaining connected to the render. Currently a snapshotted image seems to contain `isRendering == true` in the metadata, but that seems misleading.

Reply:
Would be good to add a test case for this one.

Reply:
Good spot thanks. Fixed [here](https://github.com/GafferHQ/gaffer/pull/4018/commits/4cd8d0234fbd19b6f8c425991dcb65b2c4f9ed02#diff-7b870efb0bea4a543d263290391461a1ab5c623cac1c13272adf5bdf6c622ec6R205).

Reply:
Test added [here](https://github.com/GafferHQ/gaffer/pull/4018/commits/4cd8d0234fbd19b6f8c425991dcb65b2c4f9ed02#diff-2de959ea3d78907f9495080b210a83440e22e8cd21b202959086f85b86e05863R551-R553).